utils/screenshot_utils.py

```
@@ -271,4 +272,217 @@ async def analyze_screenshot_from_data_url(data_url: str, window_title: str = ''
+            '/usr/share/fonts/opentype/ipafont-gothic/ipag.ttf',
+            '/usr/share/fonts/truetype/dejavu/DejaVuSans.ttf',
+        ]
+
+    import os
+    for path in candidates:
+        if os.path.isfile(path):
+            _avatar_font_path = path
+            logger.info(f"[avatar-annotation] 找到字体: {path}")
+            return path
+
+    logger.warning("[avatar-annotation] 未找到 CJK 字体，将使用 PIL 默认字体")
```

Ruff (0.15.9)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)